.github/workflows/rpc-test.yml

```diff
@@ -28,21 +30,17 @@ jobs:
         run: go mod tidy
         working-directory: code/go-nitro
 
-        # Start up a chain for the RPC server to work against
-      - name: Run anvil chain
-        run: anvil --host 0.0.0.0 --chain-id 1337 &
-        # TODO: We currently pass in a zeroed address for the adjudicator
-        # If we want to test further functionality we should deploy the adjudicator and pass the address in
-      - name: Run go-nitro RPC server
-        run: go run . -naaddress 0x0000000000000000000000000000000000000000 -config ./scripts/test-configs/alice.toml &> output.log &
+      - name: Run go-nitro RPC servers
+        run: go run ./scripts/start-rpc-servers.go &> output.log &
         working-directory: code/go-nitro
 
       - name: Install dependencies
         run: yarn
 
-      - name: Launch Address request using CLI tool and test against expectations
-        run: bash ./scripts/test-against-server.sh
-        working-directory: ./packages/nitro-rpc-client
+      - name: Run Create Channels script
+        # TODO: We could write a test specific script that creates channels and checks the results
+        run: npx ts-node ./scripts/client-runner.ts create-channels -w 300000 &> output.log
```

Comment:
Under what conditions will this cause the action to fail? Before we were doing a very basic assertion on the output of the CLI tool -- now I guess we will only fail on errors?

Reply:
> Under what conditions will this cause the action to fail? Before we were doing a very basic assertion on the output of the CLI tool -- now I guess we will only fail on errors?

We're no longer explicitly checking the result but because we're running the full `create-channels` script I still think we're getting a lot more coverage than we previously had.  Previously we were just checking if the address from the get address call was correct. Now we're creating ledger channels,virtual channels and using them to pay, and if anything breaks down doing that we should report an error.

I think if we do want to start adding assertions in, we should write a test specific script that the `client-runner` runs. It could look like `create-channels` but could also assert on the results after creating the channels.